### ISSUE TITLE
Causes Discord Crash, fixed that now, but still does not appear

### ISSUE THREAD

**Issue by Pysis868** [Skamt/BDAddons#53]
Errors / Stack traces:
```
TypeError: zustand is not a function
    at eval (ViewProfilePicture.plugin.js:72:2)
    at PluginManager.requireAddon (renderer.js:6390:7)
    at PluginManager.loadAddon (renderer.js:1939:20)
    at PluginManager.loadAddon (renderer.js:6306:25)
    at renderer.js:1832:41

ViewProfilePicture: Could not be compiled. See console for details.
    at PluginManager.requireAddon (betterdiscord/renderer.js:6395:13)
    at PluginManager.loadAddon (betterdiscord/renderer.js:1939:20)
    at PluginManager.loadAddon (betterdiscord/renderer.js:6306:25)
    at betterdiscord/renderer.js:1832:41

sentry.78a3b192437ca7804772.js:14 TypeError: Cannot read properties of undefined (reading 'Footer')
    at renderer.js:11591:178
    at S.render (web.fd59da44d25c6d91437d.js:12:2574403)
    at sm (web.fd59da44d25c6d91437d.js:12:1013268)
    at sp (web.fd59da44d25c6d91437d.js:12:1013066)
    at l (web.fd59da44d25c6d91437d.js:12:1054558)
    at oU (web.fd59da44d25c6d91437d.js:12:1036572)
    at web.fd59da44d25c6d91437d.js:12:1036438
    at ok (web.fd59da44d25c6d91437d.js:12:1036446)
    at oA (web.fd59da44d25c6d91437d.js:12:1033197)
    at oR (web.fd59da44d25c6d91437d.js:12:1033586)
    at r6 (web.fd59da44d25c6d91437d.js:12:982257)
    at web.fd59da44d25c6d91437d.js:12:1031069
```

Something involving finding a module by a string of 128? but also "useStore, api".

Environment:
```
stable 343474 (129221b)
Host 0.0.73 x64 Build Override: N/A
Linux 64-bit (6.11.6-200.fc40.x86_64)
```

**Comment by Skamt** [comment]
a crash really? it doesn't crash for me, just reports the errors

**Comment by Pysis868** [comment]
It was, at least, disabling the interface, that I could only use again after disabling the plugin and reloading Discord.

Trying the 1.0.2 version, but I don't see the icon to view a picture on the profile page in the upper right corner.

**Comment by Skamt** [comment]
Latest broken VPP version is `1.2.13`, i think you picked the wrong plugin

**Comment by Pysis868** [comment]
Oh yea, maybe got it mixed up when writing here after I had looked at a diff.
Had that version, trying 1.2.14, still no links.
```
stable 344613 (5ac3c2d)
Host 0.0.74 x64 Build Override: N/A
Linux 64-bit (6.11.6-200.fc40.x86_64)
```

**Comment by Skamt** [comment]
well i don't know what to tell you, i'm running it and it's working

**Comment by Pysis868** [comment]
Was doing all this investigation:
<details>
Checked the sample image again today to remind me of the appearance and position of the button.
https://raw.githubusercontent.com/Skamt/BDAddons/main/ViewProfilePicture/assets/preview1.png

Hovering that area does not reveal anything new or different.

I don't even see HTML code for the intended button from this plugin either..

`layer root outer inner wrapper` has the `friends` and `more` icons/buttons.
I don't even see this action in that more menu either.

`header mask fO` has the `banner` image.

Not in either of those places, or elsewhere that I can see in the header section of the profile dialog box.

Searching in the dev tools elements view with `VPP-` does not find anything.

I see the script file loaded as `betterdiscord://plugins/ViewProfilePicture.plugin.js`

No related console errors that I saw, only sentry and `/app?_=1731956791516:1 Uncaught (in promise) Error: Error invoking remote method 'bd-get-accent-color': TypeError: electron__WEBPACK_IMPORTED_MODULE_1__.systemPreferences.getAccentColor is not a function`, and that function does not seem to be used in your script.



Updated again and restarted the app completely today:
```
stable 345368 (24ebc65)
Host 0.0.75 x64 Build Override: N/A
Linux 64-bit (6.11.6-200.fc40.x86_64)
```
</details>
when I thought, it crashed, I was shuffling plugin files around to find the one causing the error, the setting to disable them must have been toggled on, and then persisted, even when I added them back....
